packages/action-button/stories/index.ts

```diff
@@ -16,7 +16,8 @@ import '@spectrum-web-components/action-group/sp-action-group.js';
 import '@spectrum-web-components/icon/sp-icon.js';
 import '@spectrum-web-components/icons-workflow/icons/sp-icon-edit.js';
 
-import '@spectrum-web-components/action-button/sp-action-button.js';
+// Use dynamic imports otherwise the registered theme fragments doesn't run in time
+import('@spectrum-web-components/action-button/sp-action-button.js');
```

Comment:
Since styles get finalized at class declaration time, we have to delay importing ActionButton at all until we've registered our theme fragments, otherwise they have no effect.

The idea is that users would first import `Theme` and call `Theme. registerComponentFragment(componentName, css)` for all of their component fragments before importing the actual components.

I'm a bit iffy on how that would actually work, more exploration is required.